design/ceph/ceph-nfs-ganesha.md

```diff
@@ -213,8 +226,125 @@ The following directories should **not** be shared between SSSD and other contai
 - `/run/dbus`: using the DBus instance from the sidecar caused SSSD errors in testing. SSSD only
   uses DBus for internal communications and creates its own socket as needed.
 
+### Kerberos
+**NOTE:** The principles behind this design have not been validated.
+
+#### Volumes
+Volumes that should be mounted into nfs-ganesha container to support Kerberos:
+1. `keytabFile` volume: use `subPath` on the mount to add the `krb5.keytab` file to `/etc/krb5.keytab`
+2. `configFiles` volume: mount (without `subPath`) to `/etc/krb5.conf.rook/` to allow all files to
+   be mounted (e.g., if a ConfigMap has multiple data items or hostPath has multiple conf.d files)
+
+#### Ganesha config
+Should add configuration. Docs say Active_krb 5 is default true if krb support is compiled in, but
+most examples have this explicitly set.
+Default PrincipalName is "nfs".
+Default for keytab path is reportedly empty. Rook can use `/etc/ker
```

Comment:
is this the expected path or typo?
```suggestion
Default for keytab path is reportedly empty. Rook can use `/etc/ker`
```

Reply:
I assume a typo, `/etc/krb5.keytab` is the default